README.md

```diff
@@ -89,77 +89,92 @@ WSO2 API Kubernetes Platform has released following docker images in the WSO2 pu
+    >
+    > To deploy data plane components only use `--set wso2.apk.cp.enabled=false`
+
 5. Verify the deployment by executing ```kubectl get pods -n apk```
 
 ### To Access Deployment through local machine
 
-#### Identify the router-service external IP address to invoke the API through the APK gateway.
-``` kubectl get svc -n apk | grep router-service ```
-
-``` kubectl port-forward svc/apk-test-wso2-apk-router-service -n apk 9095:9095 ```
+- You can either, identify the router-service external IP address to invoke the API through the APK gateway
+    ```bash
+    kubectl get svc -n apk | grep router-service
+    ```
+- or, port forward to router-service to use localhost.
+    ```bash
+    kubectl port-forward svc/apk-test-wso2-apk-router-service -n apk 9095:9095
+    ```
 
 
 ## Quick Start APK with Kubernetes client
-Follow the instruction below to deploy an API using the kubectl.
+Follow the instruction below to deploy an API using the `kubectl`.
 
-1. Create API CR and create production and/or sandbox HTTPRoute CRs, and service for the API backend . 
-   You can find a sample CR set in `developer/tryout/samples/` folder in this repository.
+1. Create API CR and create production and/or sandbox HTTPRoute CRs, and service for the API backend. You can find a sample CR set in `developer/tryout/samples/` folder in this repository.
 
 2. Apply CRs to kubernetes API server using the kubectl.
-  ```bash
-  kubectl apply -f developer/tryout/samples/
-  ```
-Note: Services should be created in a different namespace than APK or Kubernetes System namespaces.
-Note: APIs should be created in the APK deployment namespace.
-
-3. Get a  token to invoke the System API. Provide the router service external ip to below command.
-  ```bash
-  TOKEN=$(curl --location --request POST '{router_service}:9095/oauth2/token' \
---header 'Host: idp.am.wso2.com' \
---header 'Authorization: Basic NDVmMWM1YzgtYTkyZS0xMWVkLWFmYTEtMDI0MmFjMTIwMDAyOjRmYmQ2MmVjLWE5MmUtMTFlZC1hZmExLTAyNDJhYzEyMDAwMg==' \
---header 'Content-Type: application/x-www-form-urlencoded' \
---data-urlencode 'grant_type=client_credentials')
-  ```
+    ```bash
+    kubectl apply -f developer/tryout/samples/ -n apk
+    ```
+    > **Note**
+    >
+    > Services should be created in a different namespace than APK or Kubernetes System namespaces.
+    >
+    > APIs should be created in the APK deployment namespace.
+    >
+    > Provide the router service external ip to `{router_service}` in below commands.
+
+3. Get a token to invoke the System API.
+    ```bash
+    ACCESS_TOKEN=$(curl --location --request POST '{router_service}:9095/oauth2/token' \
+    --header 'Host: idp.am.wso2.com' \
+    --header 'Authorization: Basic NDVmMWM1YzgtYTkyZS0xMWVkLWFmYTEtMDI0MmFjMTIwMDAyOjRmYmQ2MmVjLWE5MmUtMTFlZC1hZmExLTAyNDJhYzEyMDAwMg==' \
+    --header 'Content-Type: application/x-www-form-urlencoded' \
+    --data-urlencode 'grant_type=client_credentials' | jq -r '.access_token')
```

Comment:
this command implies, that it requires the jq, isn't it?

Reply:
yes